Changelog.md

```diff
@@ -64,6 +64,7 @@ Language Features:
  * General: Allow appending ``calldata`` keyword to types, to explicitly specify data location for arguments of external functions.
  * General: Support ``pop()`` for storage arrays.
  * General: Scoping rules now follow the C99-style.
+ * Type Checker: Disallow memory and storage structs in events without the experimental feature ABIEncoderV2.
```

Comment:
Is this a language feature or a bugfix?

Reply:
It's a bugfix for the crash case (non-indexed), and an error reporting for the indexed case where the compiler didn't complain.
I guess in the end it is only a bugfix